### ISSUE TITLE
Strange issue with 200S and adjusting the speed of the unit

### ISSUE THREAD

**Issue by d3adc0de** [RaresAil/homebridge-levoit-air-purifier#17]
### Describe The Bug

The unit has only 4 fan speeds. 25% appears to trigger night mode, anything above that is settings I, II, III on the unit.

The plugin can turn the device off and on without issue, however there seems to be an issue with adjusting the fan speed after a period of time has elapsed. To reproduce:

1. Trigger night mode (25%)
2. Wake up in the morning
3. Ask Siri to adjust the fan speed to 50% (or any %, it doesn't matter). 
4. Siri will say that the speed has been adjusted (but it hasn't)

You don't need to ask Siri. If you go in to the home app manually and attempt to adjust the fan speed it will show that the setting sticks but the fan won't kick up. If you go back in, you'll see the speed is still pinned at the 25% setting.

Interestingly, the only way around this seems to be to tell siri (or set in the home app) to set the speed to 100%, after which time you will be able to use siri (or the home app) to adjust between the 4 fan speeds no problem.

My workaround: I created a shortcut that pushes the fan speed to 100% then down to the speed I want. in sequence. I have Siri run that/run it on an automation and it works.

In my testing this issue only seems to occur when the device has been in sleep mode. It does not seem to be a problem going from a higher setting down to sleep mode.

### Expected behavior

The fan speed adjusts no matter how much time has elapsed when between adjustments from sleep mode.

### Logs

I don't have these at the moment. Happy to gather whatever is necessary.

### Plugin Config
```
{
[...SNIP...]
        {
            "name": "Levoit Air Purifiers",
            "email": "my email here",
            "password": "password is here",
            "platform": "LevoitAirPurifiers",
            "_bridge": {
                "username": "stuff is here",
                "port": 38665
            }
        }
    ]
}
```
### Plugin Version

1.1.6

### Homebridge Version

1.3.8? UI is v4.41.5

### Node.js Version

16.13.1

### NPM Version

8.1.2

### Operating System

`uname -a: 4.4.180+ #42218 SMP Mon Oct 18 19:17:56 CST 2021 x86_64 GNU/Linux`

**Comment by d3adc0de** [comment]
I've enabled debug mode to gather logs. I suspect that's going to be required. I don't see in the docs where the plugin logs to, however.

**Comment by RaresAil** [comment]
Hello, thank you for reporting this bug 😃, if you can reproduce with the debug mode it will be great, and i will start to investigate this on 28 December because now i am not home for Christmas.

**Comment by d3adc0de** [comment]
Will do! Happy holidays! 😃

**Comment by RaresAil** [comment]
Happy holidays to you too! 😃

**Comment by RaresAil** [comment]
Hey @d3adc0de , I can’t replicate this issue, are you still having this issue?

**Comment by d3adc0de** [comment]
Oddly enough it's now just working. I'll keep an eye on it and refile if it crops up again.

**Comment by RaresAil** [comment]
Alright, at that time maybe it was an issue with the VeSync Servers that caused that issue

**Comment by d3adc0de** [comment]
ok, reopening. it's doing it again.

**Comment by d3adc0de** [comment]
So this is the relevant logs:

```
 [Levoit Air Purifiers] [DEBUG]: [LOGIN] Logging in...
 [Levoit Air Purifiers] [DEBUG]: [LOGIN] The authentication success
 [Levoit Air Purifiers] [DEBUG]: [GET DEVICE INFO] Getting device info...
 [Levoit Air Purifiers] [DEBUG]: [GET DEVICE INFO] Getting device info...
 ```
 
 the "getting device info" part showed up after I said "hey siri, set air purifier to 50%" (twice). I've double checked in the Vesync app and I see it's still in "sleep" mode.

Next, I tell siri it to set to 75%. and this is what happens:

```
 [Levoit Air Purifiers] [DEBUG]: [GET DEVICE INFO] Getting device info...
 [Levoit Air Purifiers] [DEBUG]: [SEND COMMAND] Sending command setLevel to Purify Me with ({"level":2,"type":"wind","id":0})...
```
And we can see it worked. Subsequently, I tell siri to set it to 50% (from 75%) and it works:

```
 [Levoit Air Purifiers] [DEBUG]: [GET DEVICE INFO] Getting device info...
 [Levoit Air Purifiers] [DEBUG]: [SEND COMMAND] Sending command setLevel to Purify Me with ({"level":1,"type":"wind","id":0})...
```

so it seems that when it's in sleep mode, telling siri to set it from sleep -> 50% triggers this, but higher than 50% and it works?

**Comment by RaresAil** [comment]
@d3adc0de This issue should be fixed in [1.1.7](https://github.com/RaresAil/homebridge-levoit-air-purifier/releases/tag/1.1.7)
